SampleApp/SecondViewController.swift

```diff
@@ -108,5 +109,30 @@ class SecondViewController: UIViewController, UITextFieldDelegate, CLLocationMan
   func locationManager(manager: CLLocationManager, didFailWithError error: NSError) {
     print(error)
   }
+  
+  //Mapview Delegate
```

Comment:
nit: "// MARK: - Mapview Delegate" might be more useful since you can use it in the class overview view